src/terrain/util/nats.clj

```diff
@@ -46,16 +39,12 @@
 
 (defn- json-encode
   [o]
-  (json/generate-string o {:key-fn encode-key}))
+  (.print (JsonFormat/printer) o))
```

Comment:
Cheshire was throwing an error for the protocol buffers that are being created now. I suspect that the reason for this is that we're now using actual protocol buffers objects rather than the `ProtoMap` objects generated by [pronto](https://github.com/AppsFlyer/pronto).